app/views/partials/_how-it-works.html.haml

```diff
@@ -0,0 +1,20 @@
+%section.how-it-works
+  .container.how-it-works__container
+    .how-it-works__inf
+      %h1 Es muy sencillo
+
+      .how-it-works__category
+        .how-it-works__description
+          = image_tag('login.png', alt: 'login')
+          %h3 Captura
+          %Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+
+        .how-it-works__description
+          = image_tag('login.png', alt: 'login')
+          %h3 Captura
+          %Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+
+        .how-it-works__description
+          = image_tag('login.png', alt: 'login')
+          %h3 Captura
+          %Lorem ipsum dolor sit amet, consectetur adipisicing elit.
```

Comment:
Is `Lorem` and HTML tag? I think it is not. This should be a `%p` tag. What do you think?